docs/3.0/fields/radio.md

```diff
@@ -0,0 +1,52 @@
+---
+version: '3.15.0'
+license: community
+---
+
+# Radio
+
+<Image src="/assets/img/fields/radio.png" width="425" height="87" alt="Radio field" />
+
+The `Radio` field is used to render radio buttons. It's useful when only one value can be selected in a given options group.
+
+### Field declaration example
+Below is an example of declaring a `radio` field for a role:
+
+```ruby
+field :role,
+  as: :radio,
+  name: "User role",
+  options: {
+    admin: "Administrator",
+    manager: "Manager",
+    writer: "Writer"
+  }
+```
+
+<Option name="`options`">
+
+A `Hash` representing the options that should be displayed in the radio buttons.
+
+The keys represent the value that will be persisted and the values are the visible labels.
+
+#### Default value
+
+Empty `Hash`.
+
+```ruby
+{}
+```
+
+#### Possible values
+
+Any `Hash`. Example:
+
+```ruby
+{
+  admin: "Administrator",
+  manager: "Manager",
+  writer: "Writer"
+}
+```
+
+</Option>
```

Comment:
Let's mention about the options being executed within `Avo::ExecutionContext` and that options can be passed as a Proc with access to resource record etc... Like here https://docs.avohq.io/3.0/actions.html#actions-authorization

Reply:
@thiagoyoussef and @Paul-Bob.

Avo follows [Rails' way of declaring options](https://apidock.com/rails/ActionView/Helpers/FormOptionsHelper/options_for_select) where the label is the key of the hash and the id is the value of the hash.
I know, I don't agree with it, but it is what it is.

Can you please check and see that we use the same way going forward on select and radio fields?

Quite thought: Maybe we switch them in Avo 4?

Reply:
You're right. Let's leave it like that.